From f16020003d90d4070077ac944b121f4b0b7ada04 Mon Sep 17 00:00:00 2001
From: Isaiah Norton <isaiah@tiledb.io>
Date: Wed, 13 Nov 2019 13:53:45 -0500
Subject: [PATCH] Fix curl find_package by removing stray 'CONFIG'

This was left over from the curl/cmake iteration, and breaks this initial find_package, which causes curl to fallback to preferring the static library.
---
 cmake/Modules/FindCurl_EP.cmake | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/cmake/Modules/FindCurl_EP.cmake b/cmake/Modules/FindCurl_EP.cmake
index 574fffd1..2ff7d4d2 100644
--- a/cmake/Modules/FindCurl_EP.cmake
+++ b/cmake/Modules/FindCurl_EP.cmake
@@ -42,8 +42,7 @@ set(CURL_PATHS ${TILEDB_EP_INSTALL_PREFIX})
 
 # First try the CMake-provided find script.
 find_package(CURL
-  HINTS "${CURL_PATHS}"
-  QUIET ${TILEDB_DEPS_NO_DEFAULT_PATH} CONFIG)
+  QUIET ${TILEDB_DEPS_NO_DEFAULT_PATH})
 
 # Next try finding the superbuild external project
 if (NOT CURL_FOUND)
-- 
2.20.1

